### PR TITLE
[bitnami/redis] Allow podSelector from any namespaceSelector

### PR DESCRIPTION
**Pull request by mwithheld**
Issue #12607 Allow podSelector from any namespaceSelector

Signed-off-by: mwithheld <796986+mwithheld@users.noreply.github.com>

### Applicable issues

  - fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by rafariossaa** [comment]
Hi,
  Could you take a look to the errors reported by CI/CD ?

**Comment by rafariossaa** [comment]
Sorry for the delay.
It seems you need to update it again.

File: bitnami/redis/templates/networkpolicy.yaml [hunk 1]
```diff
@@ -52,11 +52,15 @@ spec:
               {{ template "common.names.fullname" . }}-client: "true"
         - podSelector:
             matchLabels: {{- include "common.labels.matchLabels" . | nindent 14 }}
-        {{- if .Values.networkPolicy.ingressNSMatchLabels }}
+        {{- if or .Values.networkPolicy.ingressNSMatchLabels .Values.networkPolicy.ingressNSPodMatchLabels }}
         - namespaceSelector:
             matchLabels:
-              {{- range $key, $value := .Values.networkPolicy.ingressNSMatchLabels }}
-              {{ $key | quote }}: {{ $value | quote }}
+              {{- if or .Values.networkPolicy.ingressNSMatchLabels }}
```
Comment (carrodher) [inline bitnami/redis/templates/networkpolicy.yaml:58]: Here there is a `or` directive but only one value, could you please check it?

Comment (mwithheld) [reply bitnami/redis/templates/networkpolicy.yaml]: Good catch -- fixed.

**Review by rafariossaa — APPROVED** [review]
LGTM
Thanks for adding this

**Pull request merged**